src/routes/Wallets/routes/Create/sagas/walletsCreate.js

```diff
@@ -6,117 +6,96 @@ import { put, select, takeEvery } from 'redux-saga/effects'
 import config from 'config'
 import walletsWorker from 'workers/wallets'
 import { selectWallets, selectWalletsCreate } from 'store/stateSelectors'
+import * as wallets from 'routes/Wallets/modules/wallets'
 
-import { setWallets, setActiveWallet } from 'routes/Wallets/modules/wallets'
-import type { WalletsSetWalletsAction } from 'routes/Wallets/modules/wallets'
-
-import {
-  STEPS,
-  OPEN_VIEW,
-  CLOSE_VIEW,
-  CREATE_ERROR,
-  CREATE_SUCCESS,
-  GO_TO_NEXT_STEP,
-  GO_TO_PREV_STEP,
-  CHECK_NAME_ERROR,
-  CHECK_NAME_SUCCESS,
-  clean,
-  goToStartView,
-  setCurrentStep,
-  setInvalidField,
-} from '../modules/walletsCreate'
-
-function* checkName(): Saga<void> {
-  const { items }: WalletsState = yield select(selectWallets)
-  const { name }: WalletsCreateState = yield select(selectWalletsCreate)
-
-  if (!name) {
-    yield put(setInvalidField('name', 'Name should not be empty'))
-
-    return
-  }
-
-  walletsWorker.checkNameRequest(items, name)
-}
-
-function* checkNameError(action: { payload: Error }): Saga<void> {
-  yield put(setInvalidField('name', action.payload.message))
-}
-
-function* checkNameSuccess(): Saga<void> {
-  yield put(setCurrentStep(STEPS.PASSWORD))
-}
+import * as walletsCreate from '../modules/walletsCreate'
 
-function* create(): Saga<void> {
-  const walletsData: WalletsState = yield select(selectWallets)
+function* createWallet(): Saga<void> {
+  const walletsData = yield select(selectWallets)
 
   const {
     name,
     password,
     passwordHint,
     passwordConfirm,
-  }: WalletsCreateState = yield select(selectWalletsCreate)
+    testPasswordData,
+  }: WalletsState = walletsData
 
-  const isPasswordExists: boolean = !!walletsData.testPasswordData
+  const isPasswordExists: boolean = !!testPasswordData
 
   if (!isPasswordExists) {
+    if (password === name) {
```

Comment:
optional: use name.trim() to remove unused spaces

Reply:
we have JWWEB-159 for that

Reply:
well if its one line, better to fix it now, instead of creating a new ticket.

Reply:
The issue has already created. And we must to discuss little bit more with designer & PO about behaviour of this field